tests/system/serviceScale_test.go

```diff
@@ -18,18 +17,30 @@ func TestServiceScale(t *testing.T) {
 	defer s.Terraform.Destroy()
 
 	serviceID := s.Terraform.Output("service_id")
+	deployID := s.Terraform.Output("deploy_id")
+
+	s.Layer0.UpdateService(serviceID, deployID, 3)
+	log.Printf("[DEBUG] Waiting for service to scale up")
+	service := s.Layer0.ReadService(serviceID)
+	for start := time.Now(); time.Since(start) < time.Minute*5; time.Sleep(time.Second * 10) {
```

Comment:
It seems like there is logical inverse bug introduced everytime `logutils.WaitFor` was replaced. Let's just move that helper function to this package and not change the internal logic. I'm going to stop commenting on them for the rest of the pr to save time.